src/core/model/Layer.h

```diff
@@ -52,6 +52,11 @@ class Layer {
      */
     ElementIndex removeElement(Element* e, bool free);
 
+    /**
+     * Removes all Element%s from the Layer
+     */
+    void clear();
```

Comment:
The fact it does not delete the elements must be abundantly clear

```suggestion
     * Removes all Elements from the Layer *without freeing them*
     */
    void clearNoFree();
```